queries/slippage/subquery_batchwise_internal_transfers.sql

```diff
@@ -166,6 +171,17 @@ valued_potential_buffered_trades as (
                              on pusd.contract_address = t.token
                                  and date_trunc('minute', block_time) = pusd.minute
 ),
+internal_buffer_trader_solvers as (
+    Select 
+    CONCAT('0x', ENCODE(address, 'hex')) 
+    from gnosis_protocol_v2."view_solvers" 
+    where 
+        name = 'DexCowAgg' or
+        name = 'CowDexAg' or 
+        name = 'MIP' or
+        name = 'Quasimodo' or
+        name = 'QuasiModo'
```

Comment:
Do we have a typo in one of the solver names? It would be nice if this were fixed at the source instead of worked around here.

Reply:
Yes, we do.
Yes of course it should be fixed at the source, but I wanna do a payout today. 

And getting another PR review from dune will take time. I think it needs to be fixed here:
https://github.com/duneanalytics/abstractions/pull/862/files